aten/src/ATen/native/quantized/cudnn/Conv.cpp

```diff
@@ -199,6 +215,22 @@ void raw_cudnn_convolution_forward_out(
   // TODO: compile empty & fill_ using full_like or full
   Tensor requantize_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
   requantize_multiplier_tensor.fill_(requantize_multiplier);
+  c10::optional<at::Tensor> bias_multiplier_tensor;
+  c10::optional<at::Tensor> after_scales_bias;
+  c10::optional<at::Tensor> after_add;
+  c10::optional<at::Tensor> broadcasted_bias;
+  if (bias.has_value()) {
+    std::vector<int64_t> new_size(quantized_output.sizes().size() - 1, 1);
+    new_size[0] = bias.value().size(0);
+    broadcasted_bias = bias.value().reshape(new_size);
+    broadcasted_bias.value() = broadcasted_bias.value().broadcast_to(quantized_output.sizes());
+    broadcasted_bias.value() = broadcasted_bias.value().contiguous(c10::MemoryFormat::ChannelsLast);
+    bias_multiplier_tensor = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
+    bias_multiplier_tensor.value().fill_(bias_multiplier);
+    after_scales_bias = at::empty(quantized_output.sizes(), at::device(at::kCUDA).dtype(at::kFloat), at::MemoryFormat::ChannelsLast);
```

Comment:
btw, have we asked cudnn team about the support for inplace operations?

Reply:
saw this in the comments below

Reply:
I haven't asked yet, and it doesn't look like it was in that doc you created either. I'll check with them the next time we email them

Reply:
sg, let's keep a list of questions that we want to ask in the doc as well, just to keep track of everything